index.js

```diff
@@ -74,8 +73,12 @@ class DHT extends EventEmitter {
   static bootstrapper (port, host, opts) {
     if (!port) throw new Error('Port is required')
     if (!host) throw new Error('Host is required')
-    const id = peer.id(host, port)
-    return new this({ port, id, ephemeral: false, firewalled: false, anyPort: false, bootstrap: [], ...opts })
+    if (host === '0.0.0.0' || host === '::') throw new Error('Invalid host')
+    if (!UDX.isIPv4(host)) throw new Error('Host must be a IPv4 address')
+
+    const dht = new this({ port, ephemeral: false, firewalled: false, anyPort: false, bootstrap: [], ...opts })
+    dht._nat.add(host, port)
```

Comment:
This line alone is the fix, the `bootstrapper` method already forces you to pass the actual globally accessible host, so we can just add it to our own NAT which makes us stable / persistent